JS-condition/js-conditions-and-booleans_teenager-check/js/logType.js

```diff
@@ -0,0 +1,50 @@
+let data;
+
+let typeOfData = typeof data;
+
+let message = "";
+
+switch (typeOfData) {
+  case "undefined":
+    message = "Undefine!";
+    break;
+
+  case "string":
+    message = "String!";
+    break;
+  case "bigInt":
+    message = "BigInt!";
+    break;
+  case "function":
+    message = "Function!";
+    break;
+  case "boolean":
+    message = "boolean!";
+    break;
+
+  //number & NaN
+
```

Comment:
Don't think that's necessary. It's pretty explicit from the code that you'll be testing for number and NaN. : )

Reply:
will be remved